snaps_openstack/utilities/var.yaml

```diff
@@ -60,3 +60,16 @@ TASKS:
           -
             port_name: 
             size: ''
+  -
+   name: pre-post
+   HOSTS:
+     - 
+       ip: ""
+       node_type: "controller"
+       primary_interface: ""
+       target_interface: "veth1"
+     - 
+       ip: ""
+       primary_interface: ""
+       target_interface: "veth1"
+       node_type: "compute"
```

Comment:
Move the node_type line before primary_interface to be consistent with the above block, i.e.,
...
node_type: "compute"
primary_interface: ""
...